kotlin-styled-next/build.gradle.kts

```diff
@@ -1,15 +1,104 @@
 plugins {
-    kotlin("js")
+    kotlin("multiplatform")
     `publish-conventions`
 }
 
-dependencies {
-    api(project(":kotlin-extensions"))
-    api(project(":kotlin-css"))
-    api(project(":kotlin-react"))
-    api(project(":kotlin-react-dom"))
+repositories {
```

Comment:
Not sure what all these repositories (except mavenCentral) are needed for, seems like this section can be dropped.